#!/usr/bin/env python2
PACKAGE = "pcl_editor"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("x", double_t, 0, "x", 0, -100, 100)
gen.add("y", double_t, 0, "y", 0, -100, 100)
gen.add("z", double_t, 0, "z", 0, -100, 100)
gen.add("roll", double_t, 0, "roll", 0, -5, 5)
gen.add("pitch", double_t, 0, "pitch", 0, -5, 5)
gen.add("yaw", double_t, 0, "yaw", 0, -5, 5)
gen.add("pcd_merged", str_t, 1, "merged pcd file", "test_pcd.pcd")
gen.add("save", bool_t, 1, "save to pcd", False)

exit(gen.generate(PACKAGE, "pcl_editor", "MergerDyn"))

